### PR TITLE
Add steps required for activity app

### PR DESCRIPTION
**Pull request by paurakhsharma**
## Description
Add step required for PR https://github.com/owncloud/activity/pull/689

## Related Issue
https://github.com/owncloud/activity/issues/677

## How Has This Been Tested?
:robot: 

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Acceptance test

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have read the **CONTRIBUTING** document.
- [ ] I have added tests to cover my changes.
- [x] All new and existing tests passed.

### CODE REVIEW

**Review by skshetry — COMMENTED** [review]
:+1:

**Comment by dpakach** [comment]
:+1:

**Review by dpakach — COMMENTED** [review]
:+1:

**Review by individual-it — APPROVED** [review]

File: tests/acceptance/features/bootstrap/PublicWebDavContext.php [hunk 1]
```diff
@@ -64,7 +64,8 @@ public function downloadPublicFileWithRange($range) {
 	 * @return void
 	 */
 	public function downloadPublicFileInsideAFolderWithRange($path, $range) {
-		$fullUrl = $this->featureContext->getBaseUrl() . "/public.php/webdav$path";
+		$path = $path = \ltrim($path, "/");
```
Comment (phil-davis) [inline tests/acceptance/features/bootstrap/PublicWebDavContext.php]: why do we have 2 ``$path =`` ?

Comment (paurakhsharma) [reply tests/acceptance/features/bootstrap/PublicWebDavContext.php]: Oh, that's a mistake.

**Review by phil-davis — APPROVED** [review]

**Pull request merged**